srcpkgs/python3-gnupg/patches/0001-set-daemon-attribute-instead-of-setDaemon.patch

```diff
@@ -0,0 +1,50 @@
+From 6fb3874786d04bc05cbf6f1a1073d698b9d447f2 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Jan=20Christian=20Gr=C3=BCnhage?=
+ <jan.christian@gruenhage.xyz>
+Date: Tue, 22 Mar 2022 10:39:38 +0100
+Subject: [PATCH]  set daemon attribute, instead of setDaemon()
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+> DeprecationWarning: setDaemon() is deprecated, set the daemon
+> attribute instead
+
+Signed-off-by: Jan Christian Grünhage <jan.christian@gruenhage.xyz>
+---
```

Comment:
patch already submitted upstream as https://github.com/vsajip/python-gnupg/pull/174.

This doesn't make any actual difference, it's just been putting tons and tons of warnings out when running the tests so I patched it.

Reply:
Similar patch was accepted upstream and is included in the version used now.